internal/credential/vault/fields.go

```diff
@@ -18,5 +18,7 @@ const (
 	tlsSkipVerifyField  = "TlsSkipVerify"
 	tokenField          = "Token"
 
-	mappingOverrideField = "MappingOverride"
+	// MappingOverrideField represents the field mask indicating a mapping override
+	// update has been requested.
+	MappingOverrideField = "MappingOverride"
```

Comment:
Why does this need to be exported but not the other fields?

Reply:
I needed this value in the service because I am setting this field mask manually once a mapping change is detected based on the API request.  The rest of the fields are translated via the `this: that:` tag in the proto using the mask manager `translate` function.

https://github.com/hashicorp/boundary/blob/main/internal/proto/local/controller/storage/credential/vault/store/v1/vault.proto#L49